mne_bids/read.py

```diff
@@ -88,12 +88,15 @@ def _handle_participants_reading(participants_fname, raw,
         if infokey == 'sex':
             value = _convert_sex_options(infovalue[row_ind],
                                          fro='bids', to='mne')
+            if value is None:
+                value = 0
```

Comment:
why do you need to do this?

Reply:
To set the subject_info to 0 = unknown? 

If not, we can stick w/ setting it to None  then? what is your opinion?

Reply:
See my suggestion at https://github.com/mne-tools/mne-bids/pull/397/files#r416338210